@xen-orchestra/web/src/pages/vm/new.vue

```diff
@@ -97,31 +97,27 @@
                   <UiInput v-model="vmState.name" accent="brand" />
                 </VtsInputWrapper>
                 <!-- <UiInput v-model="vmState.tags" :label-icon="faTags" accent="brand" :label=" t('tags')" /> -->
-                <!--              <VtsInputWrapper :label="t('boot-firmware')"> -->
-                <!--                <FormSelect v-model="vmState.boot_firmware"> -->
-                <!--                  <option v-for="boot in bootFirmwares" :key="boot" :value="boot"> -->
-                <!--                    {{ boot === undefined ? t('bios-default') : boot }} -->
-                <!--                  </option> -->
-                <!--                </FormSelect> -->
-                <!--              </VtsInputWrapper> -->
-                <!--              <div -->
-                <!--                v-tooltip=" -->
-                <!--                  vmState.boot_firmware === 'uefi' || templateHasBiosStrings -->
-                <!--                    ? { -->
-                <!--                        placement: 'top-start', -->
-                <!--                        content: vmState.boot_firmware !== 'uefi' ? t('boot-firmware-bios') : t('boot-firmware-uefi'), -->
-                <!--                      } -->
-                <!--                    : undefined -->
-                <!--                " -->
-                <!--              > -->
-                <!--                <UiCheckbox -->
-                <!--                  v-model="copyHostBiosStrings" -->
-                <!--                  accent="brand" -->
-                <!--                  :disabled="vmState.boot_firmware === 'uefi' || templateHasBiosStrings" -->
-                <!--                > -->
-                <!--                  {{ t('copy-host') }} -->
-                <!--                </UiCheckbox> -->
-                <!--              </div> -->
+                <VtsInputWrapper :label="t('boot-firmware')">
+                  <VtsSelect :id="bootFirmwareSelectId" accent="brand" />
+                </VtsInputWrapper>
+                <div
+                  v-tooltip="
```

Comment:
The tooltip directive import is missing in the script section:
`import { vTooltip } from '@core/directives/tooltip.directive'`